src/UmbracoIdentity/IdentityEnabledMembersMembershipProvider.cs

```diff
@@ -10,6 +12,34 @@ namespace UmbracoIdentity
     /// </summary>
     public class IdentityEnabledMembersMembershipProvider : MembersMembershipProvider
     {
+        /// <summary>
+        /// Gets a value indicating whether member usernames are only alphanumeric.
+        /// </summary>
+        /// <value></value>
+        /// <returns>true if a usernames are only alphanumeric; otherwise, false. The default is false.</returns>
+        public bool AllowOnlyAlphanumericUserNames { get; private set; }
```

Comment:
Does it make sense to have changes made to the membership provider and expose these properties? I would have instead just use your logic from within the Initialize method and put that logic inside the `UmbracoMembersUserManager` code when initializing both the `UserValidator` and `PasswordValidator`. But what do you think?

Reply:
I put them there to mimic what Umbraco does in it's Umbraco.Core.Security.MembershipProviderBase, and that's the first place we inherit from it. So I thought it was fairly neat, extending the existing properties with similar visibility of them?